src/components/media/SearchMedia.tsx

```diff
@@ -34,58 +39,108 @@ const SearchMedia = (): ReactElement => {
-      <div className="search-layout">
-        <div className="logo">
-          <KitsuDatabaseTools />
-        </div>
-        <AddEntryButton className="new-entry">Add New Entry</AddEntryButton>
-
-        <div className="media-type-dropdown">
-          <MediaDropdown />
-        </div>
-        <div className="searchbox">
-          <label htmlFor="media-searchbar">
-            Search for selected media type
-          </label>
-          <Input
-            id="media-searchbar"
-            type="search"
-            placeholder="Search"
-            autoComplete="off"
-            onChange={(e) => {
-              setSearchTitle(e.target.value);
+    <>
+      <SearchMediaLayout>
+        <div className="search-layout">
+          <div className="logo">
+            <KitsuDatabaseTools />
+          </div>
+          <AddEntryButton className="new-entry">Add New Entry</AddEntryButton>
 
-              executeSearch({
-                variables: searchTitleVariables,
-              });
-            }}
-          />
-          <FontAwesomeIcon icon={faSearch} />
+          <div className="media-type-dropdown">
+            <MediaDropdown />
+          </div>
+          <div className="searchbox">
+            <label htmlFor="media-searchbar">
+              Search for selected media type
+            </label>
+            <Input
+              id="media-searchbar"
+              type="search"
+              placeholder="Search"
+              autoComplete="off"
+              /* onChange={(e) => {
+                setSearchTitle(e.target.value);
+              }} */
+              onChange={handleChange}
+            />
+            <FontAwesomeIcon icon={faSearch} />
+          </div>
         </div>
-
-        <div>
-          {data?.searchMediaByTitle?.nodes?.map(
-            (media: Maybe<SearchMediaFieldsFragment>) => {
+      </SearchMediaLayout>
+      <SearchResultLayout>
+        <div className="search-results">
+          {!data && loading ? (
+            <Loading></Loading>
+          ) : error ? (
+            <span className="search-error">error</span>
+          ) : data ? (
+            data?.searchMediaByTitle?.nodes?.map((media) => {
               return (
-                <Link key={media?.id} to={`/${media?.type}/${media?.id}`} />
+                <SearchResults key={media?.id}>
+                  <Poster
+                    className="poster-image"
+                    style={{
+                      backgroundImage:
+                        'url(' + media?.posterImage.original.url + ')',
+                    }}
+                  />
+                  <Link
+                    className="media-title"
+                    to={`/${media?.type}/${media?.id}`}>
+                    {media?.titles.canonical}
+                  </Link>
+                  <div className="search-description">
+                    <span>{media?.description.en}</span>
+                  </div>
+                </SearchResults>
```

Comment:
The reason why I wanted a fragment is so we can move this to another component lol.

Reply:
Don't really *need* that tho. Just pass the data through to the child.

`<SearchResult data={media} />`

I should've just just done that while I was working on it tbh